ui/raidboss/data/07-dt/raid/r11n.ts

```diff
@@ -0,0 +1,322 @@
+        sword: Outputs.intercards,
+        comboWeapons: '${first} => ${second} => ${third}',
+        unknown: Outputs.unknown,
+      },
+    },
+    {
+      id: 'R11N Dance Of Domination',
+      type: 'StartsUsing',
+      netRegex: { id: 'B3D1', source: 'The Tyrant', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'R11N Raw Steel Buster',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData['rawSteelBuster'], capture: true },
+      condition: (data, matches) => data.role === 'tank' || data.me === matches.target,
+      response: Responses.sharedTankBuster(),
+    },
+    {
+      id: 'R11N Raw Steel Spread',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData['rawSteelSpread'], capture: true },
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'R11N Charybdistopia',
+      type: 'StartsUsing',
+      netRegex: { id: 'B3D7', source: 'The Tyrant', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'R11N Ultimate Trophy Weapons Call',
+      type: 'ActorControlExtra',
+      netRegex: { category: '0197', param1: ['11D1', '11D2', '11D3'], capture: true },
+      condition: (data) => data.trophyActive,
+      delaySeconds: 2.4, // Allow for executing previous call.
+      alertText: (_data, matches, output) => {
+        const nextWeapon = weaponModelIDMap[matches.param1];
+        if (nextWeapon === 'axe')
+          return output.axe!();
+        if (nextWeapon === 'scythe')
+          return output.scythe!();
+        if (nextWeapon === 'sword')
+          return output.sword!();
+        return output.unknown!();
+      },
+      outputStrings: {
+        axe: {
+          en: 'Out next',
+        },
+        scythe: {
+          en: 'In next',
+        },
+        sword: {
+          en: 'Intercards next',
+        },
+        unknown: Outputs.unknown,
+      },
+    },
+    {
+      id: 'R11N One And Only',
+      type: 'StartsUsing',
+      netRegex: { id: 'B3DC', source: 'The Tyrant', capture: true },
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 5,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'R11N Cosmic Kiss', // Meteor towers
+      type: 'StartsUsing',
+      netRegex: { id: 'B3DE', source: 'Comet', capture: false },
+      suppressSeconds: 1,
+      response: Responses.getTowers(),
+    },
+    {
+      id: 'R11N Foregone Fatality',
+      type: 'Tether',
+      netRegex: { id: tetherData['foregoneTether'], capture: false },
+      suppressSeconds: 1,
+      alertText: (_data, _matches, output) => output.tetherBusters!(),
+      outputStrings: {
+        tetherBusters: Outputs.tetherBusters,
+      },
+    },
+    {
+      id: 'R11N Massive Meteor',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData['massiveMeteor'], capture: true },
+      condition: (data) => data.role !== 'tank',
+      response: Responses.stackMarkerOn(),
+    },
```

Comment:
These two triggers overlap for non-tanks. Does it make sense to have `R11N Foregone Fatality` only fire for tank players? (no strong opinion on this)

Also the `R11N Foregone Fatality` trigger re-fires every time the tether is passed, should probably have a 10 second suppress on it instead?

Reply:
Yeah, this is on me not playing tank, pretty much. Both suggestions make sense.

Reply:
Actually, yes, that's a much better point here. Reverted that element of the changes.

Reply:
@xiashtra what would you think of lowering the `stackMarkerOn` call to `info` for tanks? This would preserve the information as much as possible while ensuring there aren't two `alert`-level triggers on-screen at the same time.

Reply:
Maybe Foregone Fatality as info and Massive Meteor as alert for non-tanks, and vice versa for tanks.

Reply:
Of those two options, I think separating the severity levels make the most sense here.